relay-common/src/constants.rs

```diff
@@ -128,6 +128,10 @@ pub enum DataCategory {
     ///
     /// This is the category for indexed profiles that will be stored later.
     ProfileIndexed = 11,
+    /// Span
+    ///
+    /// Used for spans processed indepently of transactions.
+    Span = 12,
```

Comment:
We will need this datatype for standalone spans sooner or later, so added it here because it's a required field in `MetricSpec`.